src/python/pants/goal/run_tracker.py

```diff
@@ -235,6 +244,9 @@ def start(self, all_options: Options, run_start_time: float) -> None:
         goal_names: Tuple[str, ...] = tuple(all_options.goals)
         self._v2_goal_rule_names = goal_names
 
+        self.run_logs_file = Path(self.run_info_dir, "per-run-logs")
```

Comment:
run_info_dir is already unique per run... so unless we expect multiple logs, I don't see the need for another subdirectory.
and if we do need a subdirectory, it should just be named "logs"...

Reply:
This doesn't create a new directory, this creates a new file in the unique `run_info_dir` directory called `per-run-logs` every time. I guess there's no reason it can't be named just `logs` though.